src/libraries/System.Net.Security/src/System/Net/Security/CipherSuitesPolicy.cs

```diff
@@ -31,7 +31,7 @@ public IEnumerable<TlsCipherSuite> AllowedCipherSuites
             {
                 // This method is only useful only for diagnostic purposes so perf is not important
                 // We do not want users to be able to cast result to something they can modify
-                foreach (TlsCipherSuite cs in Pal.GetCipherSuites())
+                foreach (TlsCipherSuite cs in Pal.GetCipherSuites()!)
```

Comment:
`Pal.GetCipherSuites()` always returns null.

Reply:
>  always returns null

Did you mean non-null?

Reply:
Null, at least in the case of windows https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.Net.Security/src/System/Net/Security/CipherSuitesPolicyPal.Windows.cs#L16

Reply:
> Null, at least in the case of windows

We will never get here on Windows because the ctor will throw a PlatformNotSupportedException.